.erda/migrations/qa/20210830-test-plan-isarchived.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `dice_test_plans` ADD `is_archived` varchar(40) DEFAULT 'INPROGRESS' COMMENT 'test plan is archived';
```

Comment:
use bool or tinyint is enough. Refer to erda sql specification.

Reply:
done